libcontainer/process_linux.go

```diff
@@ -141,6 +141,16 @@ func (p *setnsProcess) execSetns() error {
 		p.cmd.Wait()
 		return newSystemErrorWithCause(err, "reading pid from init pipe")
 	}
+
+	// Clean up the zombie parent process
+	firstChildProcess, err := os.FindProcess(pid.PidFirstChild)
+	if err != nil {
```

Comment:
I'm not convinced that propagating the error here makes sense (especially since you might get `ECHILD` -- I haven't checked the Go source yet). Maybe we should do something like this

```go
if firstChildProcess, err := os.FindProcess(pid.PidFirstChild); err == nil {
	if _, err := firstChildProcess.Wait(); err != nil {
		return err
	}
}
```

Ditto for the `pid.Pid`. @mrunalp @dqminh @hqhq WDYT?